Car.js

```diff
@@ -1,26 +1,86 @@
-function Car(make, model, year, color, seats){
+function Car(make, model, year, color, seats, passengers){
   this.make = make;
   this.model = model;
   this.year = year;
   //TODO: add color, seats here
-
+  this.color = color;
+  this.seats = seats;
   this.running = false;
   this.owner = "manufacturer";
   this.previousOwners = [];
+  this.passengers = []; //first part of phase three, also added parameter above
+
+  if (passengers) {
+  	this.passengers = passengers;
+  } // added for first part of phase three
```

Comment:
Looks good. You can also use this notation to specify a default in case passengers is not specified:
```this.passengers = passengers || [];```